Source/Core/Common/Analytics.cpp

```diff
@@ -134,13 +134,15 @@ AnalyticsReporter::~AnalyticsReporter()
 
 void AnalyticsReporter::Send(AnalyticsReportBuilder&& report)
 {
+#ifdef USE_ANALYTICS
```

Comment:
No, USE_ANALYTICS is only defined when ENABLE_ANALYTICS is ON.
See https://github.com/dolphin-emu/dolphin/blob/master/CMakeLists.txt#L552